### ISSUE TITLE
question about "Training your own models"

### ISSUE THREAD

**Issue by ShiLiangfan11** [RenYang-home/OpenDVC#27]
when i train my models, some problems happen:
Operation received an exception: status: 5,message:could not creaate aview primitive descriptor,in file tensorflow/core/kernels/mkl_slice_op.cc:300
i donot know how to solve it,thank you for help.

**Comment by ShiLiangfan11** [comment]
...which was originally created as op 'flow_motion/flow_cnn_4/concat', defined at:
  File "OpenDVC_train_PSNR.py", line 45, in <module>
    flow_tensor, _, _, _, _, _ = motion.optical_flow(Y0_com, Y1_raw, batch_size, Height, Width)
  File "/home/shi/OpenDVC-master/motion.py", line 54, in optical_flow
    loss_4, flow_4 = loss(flow_3, im1_4, im2_4, 4)
  File "/home/shi/OpenDVC-master/motion.py", line 27, in loss
    res = convnet(im1_warped, im2, flow, layer)
  File "/home/shi/OpenDVC-master/motion.py", line 7, in convnet
    input = tf.concat([im1_warp, im2, flow], axis=-1)
  File "/home/shi/miniconda3/envs/Opendvc/lib/python3.6/site-packages/tensorflow/python/ops/array_ops.py", line 1124, in concat
    return gen_array_ops.concat_v2(values=values, axis=axis, name=name)
  File "/home/shi/miniconda3/envs/Opendvc/lib/python3.6/site-packages/tensorflow/python/ops/gen_array_ops.py", line 1033, in concat_v2
    "ConcatV2", values=values, axis=axis, name=name)
  File "/home/shi/miniconda3/envs/Opendvc/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/shi/miniconda3/envs/Opendvc/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py", line 488, in new_func
    return func(*args, **kwargs)
  File "/home/shi/miniconda3/envs/Opendvc/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3274, in create_op
    op_def=op_def)
  File "/home/shi/miniconda3/envs/Opendvc/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1770, in __init__
    self._traceback = tf_stack.extract_stack()

AbortedError (see above for traceback): Operation received an exception:Status: 5, message: could not create a view primitive descriptor, in file tensorflow/core/kernels/mkl_slice_op.cc:300
     [[node gradients/flow_motion/flow_cnn_4/concat_grad/Slice_2 (defined at OpenDVC_train_PSNR.py:102)  = _MklSlice[Index=DT_INT32, T=DT_FLOAT, _kernel="MklOp", _device="/job:localhost/replica:0/task:0/device:CPU:0"](gradients/flow_motion/flow_cnn_4/conv2d/Conv2D_grad/Conv2DBackpropInput, ConstantFolding/gradients/flow_motion/flow_cnn_1/concat_grad/ConcatOffset-folded-2, gradients/flow_motion/flow_cnn_4/concat_grad/Shape_2, gradients/flow_motion/flow_cnn_4/conv2d/Conv2D_grad/Conv2DBackpropInput:1, DMT/_404, DMT/_405)]]

**Comment by RenYang-home** [comment]
Is everything fine when you run the test code?

**Comment by ShiLiangfan11** [comment]
> Is every fine when you run the test code?

thank you very much,i have solved it .it  caused by conda when i installed tensorflow

**Comment by gxtbluerain** [comment]
@ShiLiangfan11 请问该问题是如何解决的？我最近也遇到了相同的问题

**Comment by gxtbluerain** [comment]
@ShiLiangfan11 How can this issue be resolved? I’ve encountered the same problem recently and look forward to your reply!
